Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_fw.nut

```diff
@@ -1686,7 +1742,7 @@ void function FW_createHarvester()
 	fw_harvesterImc.harvester.Minimap_SetZOrder( MINIMAP_Z_OBJECT )
 	fw_harvesterImc.harvester.Minimap_SetCustomState( eMinimapObject_prop_script.FD_HARVESTER )
 	AddEntityCallback_OnDamaged( fw_harvesterImc.harvester, OnHarvesterDamaged )
-	AddEntityCallback_OnPostDamaged( fw_harvesterImc.harvester, OnHarvesterPostDamaged )
```

Comment:
You cant remove the PostDamageCallback. If you want to scale damage for shield as well just move the scaling bit to the DamageCallback

Reply:
I don't think PostDamageCallback is a good idea, for if we can't pass damage from DamagedCallback, the damage will already been scaled